src/main/java/org/openhab/automation/jrule/internal/items/JRulePersistenceExtensions.java

```diff
@@ -32,6 +39,46 @@
  * @author Arne Seime - Initial contribution
  */
 class JRulePersistenceExtensions {
+    public static void persist(String itemName, ZonedDateTime timestamp, JRuleValue state, String serviceId) {
+        Item item = getItem(itemName);
+        JRuleEngine jRuleEngine = JRuleEngine.get();
+        PersistenceServiceRegistry persistenceServiceRegistry = jRuleEngine.getPersistenceServiceRegistry();
+        PersistenceService persistenceService = persistenceServiceRegistry
+                .get(Optional.ofNullable(serviceId).orElse(persistenceServiceRegistry.getDefaultId()));
```

Comment:
Should we really get the default service here? Maybe better to throw error since it is called with a specific service id.

Reply:
I did it equivalent to the query methods. I think it's more consistent.